src/Ocelot/Configuration/File/FileGlobalConfiguration.cs

```diff
@@ -42,5 +43,7 @@ public FileGlobalConfiguration()
         /// </list>
         /// </remarks>
         public string DownstreamHttpVersionPolicy { get; set; }
+
+        public FileMetadataOptions MetadataOptions { get; set; }
```

Comment:
```suggestion
        public FileMetadataOptions Metadata { get; set; }
```

Reply:
Nope, we are following the naming conventions, so @raman-m it must be `MetadataOptions`

Reply:
🆗